src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  background-color: var(--grey-color);
+  margin: 0;
+}
+
+:root {
+  --black-color: #000;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --grey-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)